taxonomy/utils.py

```diff
@@ -159,6 +187,51 @@ def get_course_metadata_fields_text(course_attrs_string, course):
     return ' '.join(filter(bool, course_attr_values)).strip()
 
 
+def get_hash(text_data: str):
+    """
+    Return hash for given text_data.
+    """
+    processed_text = text_data.replace(" ", "")
+    if not processed_text:
```

Comment:
```suggestion
    if not processed_text and processed_text != "":
```

Reply:
`if not processed_text:` should cover this case as well.

```python
>>> x = ""
>>> bool(x)
False
>>> if not x: print(True)
...
True
```